src/bootstrap/src/lib.rs

```diff
@@ -77,6 +77,8 @@ const LLD_FILE_NAMES: &[&str] = &["ld.lld", "ld64.lld", "lld-link", "wasm-ld"];
 #[allow(clippy::type_complexity)] // It's fine for hard-coded list and type is explained above.
 const EXTRA_CHECK_CFGS: &[(Option<Mode>, &str, Option<&[&'static str]>)] = &[
     (None, "bootstrap", None),
+    // #[cfg(bootstrap)] to be removed when Cargo is updated
+    (None, "test", None),
```

Comment:
Shouldn't be this a `FIXME` note?

Reply:
`#[cfg(bootstrap)]` is the fixme in this case, T-release automatically looks at them when [updating the bootstrap compiler](https://forge.rust-lang.org/release/process.html#master-bootstrap-update-tuesday).

Arguably we don't have a text associated with it, but it's less common in bootstrap to have those cfgs so I wanted to be explicit.